chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1374,7 +1374,7 @@ impl PeerManagerActor {
                        find_route_error,
                        msg,
                 );
-                trace!(target: "network", "Known peers: {:?}", self.routing_table_view.get_accounts_keys());
+                trace!(target: "network", message = "Known peers", known_peers = ?self.routing_table_view.get_accounts_keys().collect::<Vec<&AccountId>>());
```

Comment:
I don't think there's an easy way to avoid this collect. My understanding is that tracing won't evaluate the expression unless the `trace` level is actually enabled, so this is probably not important. `.collect` here also should be cheaper than the actual string formatting.

Reply:
Also, we ended up not using `message = `